### PR TITLE
Layout > Bottom sheet

### PR DESCRIPTION
**Pull request by yaraki**

### CODE REVIEW

File: Motion/app/src/main/java/com/example/android/motion/demo/bottomsheet/BottomSheetActivity.kt [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        header.setOnClickListener {
+            behavior.state = BottomSheetBehavior.STATE_EXPANDED
+        }
+        content.setOnClickListener {
+            behavior.state = BottomSheetBehavior.STATE_COLLAPSED
+        }
+
+        // Monitor movement of the bottom sheet and change the appearance of its content.
+        val evaluator = ArgbEvaluatorCompat()
+        behavior.setBottomSheetCallback(object : BottomSheetBehavior.BottomSheetCallback() {
```
Comment (ashraf-patel) [inline Motion/app/src/main/java/com/example/android/motion/demo/bottomsheet/BottomSheetActivity.kt]: `setBottomSheetCallback` is deprecated.

File: Motion/app/src/main/res/layout/bottom_sheet_activity.xml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                    style="@style/Widget.MaterialComponents.Button.TextButton"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:padding="@dimen/spacing_medium"
+                    android:text="@string/favorite"
+                    app:drawableTopCompat="@drawable/ic_favorite" />
+
+                <TextView
+                    android:id="@+id/bookmar"
```
Comment (ashraf-patel) [inline Motion/app/src/main/res/layout/bottom_sheet_activity.xml]: typo `bookmar`

Comment (yaraki) [reply Motion/app/src/main/res/layout/bottom_sheet_activity.xml]: Fixed

File: Motion/app/src/main/java/com/example/android/motion/demo/bottomsheet/BottomSheetActivity.kt (hunk 1 above)
Comment (yaraki) [reply Motion/app/src/main/java/com/example/android/motion/demo/bottomsheet/BottomSheetActivity.kt]: Fixed, thank you"!

**Pull request closed without merging**